### DISCUSSION TITLE
How to set recipe.json for multiple environments?

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#13912] [Q&A]
Is it possible to set recipe.json for Development, Release environments as in appsettings.json? If yes, how could you develop it?

**Comment by hishamco** [comment]
Import/Export feature allows you to import the data regardless of the environment. So, the question is how setting recipes for each environment will make a difference.

**Comment by ns8482e** [comment]
I believe all recipes in a module are discovered at app level. However you can have an environment variable available during recipe execution
